litex/tools/litex_json2dts_linux.py

```diff
@@ -44,18 +44,35 @@ def generate_dts(d, initrd_start=None, initrd_size=None, polling=False):
     if initrd_size is None:
         initrd_size = default_initrd_size
 
+    if initrd == "enabled" or initrd is None:
```

Comment:
Doesn't `initrd == None` intuitively mean "disabled"?

Reply:
Perhaps `initrd_size` should have priority - meaning that if it's set then `initrd_enabled` is set to `True`?

Reply:
Currently we have:
  - No args - means `--initrd enabled --initrd_start 8mb --initrd_size 8mb`
  - `--initrd disabled` - no initrd
  - `--initrd filename.xyz` means - `--initrd enabled --initrd_start 8mb --initrd_size sizeof(filename.xyz)`

I didn't want to change the defaults but we could change it to have initrd disabled by default i.e.:
  - No args -> Initrd disabled
  - `--initrd_size 8mb` -> initrd enabled
  - `--initrd filname.xyz` -> initrd enabled and `--initrd_size sizeof(filename.xyz)`

Reply:
I understand the argument about keeping the defaults.

I guess @enjoy-digital has a better intuition how destructive changing defaults/signature would be.